[2023-06-10T02:16:36.617+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: green_taxi_data.download_dataset_task scheduled__2021-11-02T06:30:00+00:00 [queued]>
[2023-06-10T02:16:36.634+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: green_taxi_data.download_dataset_task scheduled__2021-11-02T06:30:00+00:00 [queued]>
[2023-06-10T02:16:36.634+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-06-10T02:16:36.635+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2023-06-10T02:16:36.635+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-06-10T02:16:36.654+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): download_dataset_task> on 2021-11-02 06:30:00+00:00
[2023-06-10T02:16:36.658+0000] {standard_task_runner.py:54} INFO - Started process 4675 to run task
[2023-06-10T02:16:36.671+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'green_taxi_data', 'download_dataset_task', 'scheduled__2021-11-02T06:30:00+00:00', '--job-id', '228', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_multiFile_v02.py', '--cfg-path', '/tmp/tmplh6n7_a3']
[2023-06-10T02:16:36.676+0000] {standard_task_runner.py:83} INFO - Job 228: Subtask download_dataset_task
[2023-06-10T02:16:36.681+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/data_ingestion_gcs_multiFile_v02.py
[2023-06-10T02:16:36.787+0000] {data_ingestion_gcs_multiFile_v02.py:20} INFO - dtc_data_lake_dtc-dezc-001 is the bucket.
[2023-06-10T02:16:36.808+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2023-06-10T02:16:36.858+0000] {task_command.py:384} INFO - Running <TaskInstance: green_taxi_data.download_dataset_task scheduled__2021-11-02T06:30:00+00:00 [running]> on host f41d61956f54
[2023-06-10T02:16:36.884+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:204: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-06-10T02:16:36.918+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=green_taxi_data
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2021-11-02T06:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-02T06:30:00+00:00
[2023-06-10T02:16:36.919+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-10T02:16:36.920+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'curl -sSLf https://github.com/DataTalksClub/nyc-tlc-data/releases/tag/green/green_tripdata_2021-11.csv.gz > /opt/***/green_tripdata_2021-11.csv.gz']
[2023-06-10T02:16:36.931+0000] {subprocess.py:86} INFO - Output:
[2023-06-10T02:16:37.570+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-06-10T02:16:37.594+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=green_taxi_data, task_id=download_dataset_task, execution_date=20211102T063000, start_date=20230610T021636, end_date=20230610T021637
[2023-06-10T02:16:37.656+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-06-10T02:16:37.687+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-06-11T01:20:28.392+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: green_taxi_data.download_dataset_task scheduled__2021-11-02T06:30:00+00:00 [queued]>
[2023-06-11T01:20:28.400+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: green_taxi_data.download_dataset_task scheduled__2021-11-02T06:30:00+00:00 [queued]>
[2023-06-11T01:20:28.401+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-06-11T01:20:28.402+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2023-06-11T01:20:28.402+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-06-11T01:20:28.411+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): download_dataset_task> on 2021-11-02 06:30:00+00:00
[2023-06-11T01:20:28.417+0000] {standard_task_runner.py:54} INFO - Started process 5316 to run task
[2023-06-11T01:20:28.420+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'green_taxi_data', 'download_dataset_task', 'scheduled__2021-11-02T06:30:00+00:00', '--job-id', '555', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_multiFile_v02.py', '--cfg-path', '/tmp/tmpy6ls3w79']
[2023-06-11T01:20:28.421+0000] {standard_task_runner.py:83} INFO - Job 555: Subtask download_dataset_task
[2023-06-11T01:20:28.423+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/data_ingestion_gcs_multiFile_v02.py
[2023-06-11T01:20:28.489+0000] {data_ingestion_gcs_multiFile_v02.py:20} INFO - dtc_data_lake_dtc-dezc-001 is the bucket.
[2023-06-11T01:20:28.507+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2023-06-11T01:20:28.552+0000] {task_command.py:384} INFO - Running <TaskInstance: green_taxi_data.download_dataset_task scheduled__2021-11-02T06:30:00+00:00 [running]> on host f41d61956f54
[2023-06-11T01:20:28.577+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:204: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-06-11T01:20:28.610+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=green_taxi_data
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2021-11-02T06:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-02T06:30:00+00:00
[2023-06-11T01:20:28.611+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-11T01:20:28.612+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'curl -sSLf https://github.com/DataTalksClub/nyc-tlc-data/releases/download/green/green_tripdata_2021-11.csv.gz > /opt/***/green_tripdata_2021-11.csv.gz']
[2023-06-11T01:20:28.624+0000] {subprocess.py:86} INFO - Output:
[2023-06-11T01:20:28.857+0000] {subprocess.py:93} INFO - curl: (22) The requested URL returned error: 404
[2023-06-11T01:20:28.863+0000] {subprocess.py:97} INFO - Command exited with return code 22
[2023-06-11T01:20:28.877+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 22.
[2023-06-11T01:20:28.881+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=green_taxi_data, task_id=download_dataset_task, execution_date=20211102T063000, start_date=20230611T012028, end_date=20230611T012028
[2023-06-11T01:20:28.891+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 555 for task download_dataset_task (Bash command failed. The command returned a non-zero exit code 22.; 5316)
[2023-06-11T01:20:28.918+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2023-06-11T01:20:28.938+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
